project/Dependencies.scala

```diff
@@ -61,7 +61,7 @@ object Dependencies {
     // Compile
 
     val config = "com.typesafe" % "config" % "1.4.2" // ApacheV2
-    val netty = "io.netty" % "netty" % nettyVersion // ApacheV2
+    val netty = "io.netty" % "netty-all" % nettyVersion // ApacheV2
```

Comment:
I think we would prefer the needed individual netty dependencies rather than netty-all. I see that akka-persistence-r2dbc brings in individual netty dependencies and I guess it could be problematic to also mix with netty-all.

Reply:
I can do that, that would be better.